PyDSS/api/src/web/create_schema.py

```diff
@@ -9,4 +9,4 @@ def getJSONschema(ip, port, path):
     with open(path, 'w') as outfile:
         json.dump(response.json(), outfile)
 
-getJSONschema("127.0.0.1", 9090, r"C:\Users\alatif\Desktop\PyDSS_service\pydss_service\test.json")
\ No newline at end of file
+#getJSONschema("127.0.0.1", 9090, r"C:\Users\alatif\Desktop\PyDSS_service\pydss_service\test.json")
```

Comment:
Here is another case of a non-repo file.

Reply:
please remove